src/pages/LearningResources.js

```diff
@@ -1,231 +1,178 @@
+        </p>
+        <p className='text-base text-blue-700 sm:mt-4 sm:text-base md:text-base'> Got a resource you think we should add to the list?{' '}
+          <a className='underline' href={'https://twitter.com/DevProtege'}>
+            {' '}
+            Reach out!
+          </a>
+        </p>
       </div>
 
-      <div>
-        <h2 className='text-lg mb-3 text-teal-600 font-semibold'>Paid</h2>
-        <motion.ul
-          animate={{
-            y: [-5, 0],
-          }}
-          transition={{
-            delay: 0.2,
-            duration: 0.25,
-          }}
-        >
-          <li className='underline mb-3 text-blue-700 hover:text-teal-600'>
-            <a
-              href='https://egghead.io/?rc=84jyz'
-              target='_blank'
-              rel='noopener noreferrer'
-            >
-              Egghead.io
-            </a>
-          </li>
-
-          <li className='underline mb-3 text-blue-700 hover:text-teal-600'>
-            <a
-              href='https://wesbos.com/courses/'
-              target='_blank'
-              rel='noopener noreferrer'
-            >
-              Wes Bos Courses
-            </a>
-          </li>
-
-          <li className='underline mb-3 text-blue-700 hover:text-teal-600'>
-            <a
-              href='https://www.leveluptutorials.com/'
-              target='_blank'
-              rel='noopener noreferrer'
-            >
-              LevelUpTutorials
-            </a>
-          </li>
-
-          <li className='underline mb-3 text-blue-700 hover:text-teal-600'>
-            <a
-              href='https://teamtreehouse.com/'
-              target='_blank'
-              rel='noopener noreferrer'
-            >
-              Team Treehouse
-            </a>
-          </li>
-
-          <li className='underline mb-3 text-blue-700 hover:text-teal-600'>
-            <a
-              href='https://adamlearns.com/'
-              target='_blank'
-              rel='noopener noreferrer'
-            >
-              Adam Learns
-            </a>
-          </li>
-
-          <li className='underline mb-3 text-blue-700 hover:text-teal-600'>
-            <a
-              href='https://udemy.com/'
-              target='_blank'
-              rel='noopener noreferrer'
-            >
-              Udemy
-            </a>
-          </li>
-
-          <li className='underline mb-3 text-blue-700 hover:text-teal-600'>
-            <a
-              href='https://frontendmasters.com/'
-              target='_blank'
-              rel='noopener noreferrer'
-            >
-              Frontend Masters
-            </a>
-          </li>
-        </motion.ul>
+      <div className='container justify-start my-4 px-8 border-b-1'>
+        <h3 className='text-xl text-blue-900 sm:text-2xl md:text-3xl'>Free Resources</h3>
       </div>
-    </motion.div>
+
+      <div className='content-center flex grid-flow-col flex-wrap'>
```

Comment:
add class `justify-center` to be: `<div className='content-center flex grid-flow-col flex-wrap justify-center'>` so on mobile the cards will be centered.

Reply:
Adjusted Free Resources content div to be ```justify-center```

![iPhone 11_414x896_localhost_ Jul_26_03-49](https://user-images.githubusercontent.com/14058890/88487939-a8d72d80-cf57-11ea-81b0-67d9a2a4cff0.png)